BOUNTY.md

```diff
@@ -0,0 +1,116 @@
+   maintainability, design quality, and adherence to spec. Note that other developers may submit competing proposals
+   at this time as well.
+4. If your proposal is accepted, you may begin code implementation that adheres to your proposal.
+5. Submit code as a draft pull request in the appropriate repository so GraphQL.NET community leaders can track progress.
+6. Once code is complete, remove draft status and request a review from the GraphQL.NET maintainers.
+7. Update code with feedback from core GraphQL.NET contributors as needed.
+8. Wait for your code to be merged into the `master` or `develop` branch.
+9. [Submit an expense through Open Collective](https://opencollective.com/graphql-net/expenses/new) for the amount of the bounty.
+
+
+## Collaboration with GraphQL.NET team
+
+Collaboration with the GraphQL.NET team is crucial to having your code pulled into GraphQL.NET. During the proposal process,
+clarify any questions you or your team may have as early as possible. During development, use the pull request commenting
+feature.
+
+
+## Deadlines
+
+If your team does not show code commits or interaction for two weeks at a time as part of an **`in progress`** bounty, then
+the bounty will be released for another team to work on. If the team can demonstrate work on the project, they can reclaim
+the bounty.
+
+If you no longer wish to work on a bounty that has been granted to you, please notify the GraphQL.NET team.
+
+
+## Bounty Valuation
+
+The GraphQL.NET team currently uses a simple rubric to determine how to price a bounty. Generally speaking, RFPs are evaluated
+based on two main criteria:
+
+* How complex is the feature/bugfix?
+    * How long do we expect it may take to complete?
+    * How much specialized knowledge do we expect it will require to implement?
+* How high of a demand is there for this feature/bugfix?
+    * How many people do we expect this to impact?
+    * How frequently are we asked about this feature or bug?
+    * How urgently does this need to be implemented?
+
+In general, issues that are higher in complexity and higher in demand are given greater bounties.
+
+
+## Legal
+
+You ARE eligible to participate in the Program if you meet all of the following criteria:
+
+* You are 14 years of age or older. If you are at least 14 years old but are considered a minor in your place of residence,
+  you must obtain your parent's or legal guardian's permission prior to participating in this Program; and
+
+* You are either an individual researcher participating in your own individual capacity, or you work for an organization
+  that permits you to participate. You are responsible for reviewing your employer's rules for participating in this Program.
+
+You ARE NOT eligible to participate in the Program if you meet any of the following criteria:
+
+* You are a resident of any countries under U.S. sanctions (see link for current sanctions list posted by the United States
```

Comment:
What if someone wants to post a PR closing RFP and at the same time he is under sanctions? Dev **may** do it without claims to participate into program. Right? So what about other competing proposals/PRs? Will all proposals be considered on equal terms, regardless of whether a dev falls under conditions of the Program or not?

Reply:
Sure.  I mean, it's pretty explicit that the entire bounty program is at the whims of the maintainers, and we can do whatever we want.  Honestly, we don't have enough contributors to worry about it, and most of the "flowchart" is just standard practice of any outstanding issue.  I often outline a PR before posting it when I believe you may have a concern with it, or when I am looking for advice before spending time on the implementation.  Or I may post a PR, as in #3414 and then subject to discussion rewrite it like #3417 .  So here if someone says they want to work on a issue and forgo the bounty, sure we just assign it like any other.  They finish the work and we skip the payment step.  I don't think we need to write a special paragraph for this; it's common sense and we just handle the scenario if and when it arises.

Note: that's a copy and paste from Microsoft.  Probably some legal reason why it's required, just like the age thing -- and since joemcbride is located in the US and will be approving the bounties, it seemed an appropriate safeguard for his sake.  It's not like we are going to be reviewing contributor's eligibility claims.

Reply:
ok